### ISSUE TITLE
Someone copy your extension

### ISSUE THREAD

**Issue by scratchusernamemrtbts** [griffpatch/Scratch3-Dev-Tools#25]
This is the [link](https://chrome.google.com/webstore/detail/scratch-3-developer-tools/bfpdlicnalglgniemfjmbjbadekoilpl/) To the copied extension
(Is the old version)

**Comment by Explosion-Scratch** [comment]
Oof. That's bad.

**Comment by Explosion-Scratch** [comment]
lol "There are some bugs, like when you press clean up it will say griffpatch, but an update will come later" Wonder why it says griffpatch? Let's just think about that.

**Comment by griffpatch** [comment]
That's crazy!

**Comment by Joeclinton1** [comment]
From the youtube video, I'd be pretty sure the copier is just a kid, so don't go attacking him, he probably doesn't realise the consequences of what he did. I'm surprised google allowed the copy on the store though, considering their strict guidelines. A simple cease and desist will be enough to get it taken down.

**Comment by scratchusernamemrtbts** [comment]
> From the youtube video, I'd be pretty sure the copier is just a kid, so don't go attacking him, he probably doesn't realise the consequences of what he did. I'm surprised google allowed the copy on the store though, considering their strict guidelines. A simple cease and desist will be enough to get it taken down.

I already reported it

I think google chrome let him upload it because it's the old version of the extension and chrome web store didn't save the old version

**Comment by scratchusernamemrtbts** [comment]
The YouTube channel got removed

**Comment by Akash-Karthik** [comment]
i reported it from 2 accounts

**Comment by Akash-Karthik** [comment]
> From the youtube video, I'd be pretty sure the copier is just a kid, so don't go attacking him, he probably doesn't realise the consequences of what he did. I'm surprised google allowed the copy on the store though, considering their strict guidelines. A simple cease and desist will be enough to get it taken down.

But that's not the case . Anyone can do it if they **DO NOT HAVE** a brain . That guy has just cloned this repository and packed the extension . And if he is a kid i am sure he is really naughty , as he needed to pay money for just 'COPYING' and uploading an extension an extension .

**Comment by Akash-Karthik** [comment]
> > From the youtube video, I'd be pretty sure the copier is just a kid, so don't go attacking him, he probably doesn't realise the consequences of what he did. I'm surprised google allowed the copy on the store though, considering their strict guidelines. A simple cease and desist will be enough to get it taken down.
> 
> I already reported it
> 
> I think google chrome let him upload it because it's the old version of the extension and chrome web store didn't save the old version

I don't think so . He has just cloned this repository , removed the CRX file and packed it ( he must have also paid to contribute the webstore . What a waste of money ) .

**Comment by Akash-Karthik** [comment]
I contacted the developer .. i mean copier of this extension . He replied that he made some changes though , but i didnt find any . You can find those comments [here](https://chrome.google.com/webstore/detail/scratch-3-developer-tools/bfpdlicnalglgniemfjmbjbadekoilpl/support/) .

**Comment by scratchusernamemrtbts** [comment]
Why there are more user on that copied extension every week

**Comment by Akash-Karthik** [comment]
> Why there are more user on that copied extension every week

what do ya mean ?

**Comment by scratchusernamemrtbts** [comment]
I mean that last week it got 27 users but now it got 28 users
And about a month ago it got only 20 users

**Comment by Akash-Karthik** [comment]
that sucks

**Comment by scratchusernamemrtbts** [comment]
Yeah
Idk why people use it
And it already said that there is a bug that said Griffpatch said:

**Comment by Akash-Karthik** [comment]
yes i know

**Comment by scratchusernamemrtbts** [comment]
I think you should update the LICENSE file before they will be another copy of this thing

**Comment by Akash-Karthik** [comment]
i agree

**Comment by GrahamSH-LLK** [comment]
> lol "There are some bugs, like when you press clean up it will say griffpatch, but an update will come later" Wonder why it says griffpatch? Let's just think about that.

Seriously, they are trying to say that the fact that it mentions griffpatch is a bug. That's a total licence violation. Edit: The thing is, nothing else is against the LICENSE.

**Comment by scratchusernamemrtbts** [comment]
I agree
เมื่อ 17 ก.ย. 2020 13:35 เขียนโดย GrahamSH <notifications@github.com>:


lol "There are some bugs, like when you press clean up it will say griffpatch, but an update will come later" Wonder why it says griffpatch? Let's just think about that.
Seriously, they are trying to say that the fact that it mentions griffpatch is a bug. That's a total licence violation.

**Comment by scratchusernamemrtbts** [comment]
> 
> I agree
> เมื่อ 17 ก.ย. 2020 13:35 เขียนโดย GrahamSH <notifications@github.com>:
> 
> 
> lol "There are some bugs, like when you press clean up it will say griffpatch, but an update will come later" Wonder why it says griffpatch? Let's just think about that.
> Seriously, they are trying to say that the fact that it mentions griffpatch is a bug. That's a total licence violation.
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.

I reply from the iCloud web so it don't get into a dot dot dot

**Comment by scratchusernamemrtbts** [comment]
What! 35 users?![image](https://user-images.githubusercontent.com/66320961/97987775-71a5d180-1e0e-11eb-8695-38d0881f37ed.jpeg)

**Comment by Akash-Karthik** [comment]
Yeah , this is frustrating ... too frustrating ...

**Comment by Akash-Karthik** [comment]
Why didn't google take it down yet ? I have reported it from two accounts .

**Comment by LmTechyTEM** [comment]
Can you make a override folder for this for student CPU.S
